pom.xml

```diff
@@ -380,19 +385,29 @@
         </dependency>
 
         <dependency>
-            <groupId>org.vaadin</groupId>
-            <artifactId>addon-test-helpers</artifactId>
-            <version>1.10</version>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
             <scope>test</scope>
         </dependency>
-
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.10</version>
+            <groupId>org.eclipse.jetty</groupId>
```

Comment:
I think we/I should create a V8 compatible version of https://vaadin.com/directory#!addon/addon-test-helpers as well. Shitload of things to do...